release-tools/prow.sh

```diff
@@ -621,7 +621,7 @@ start_cluster () {
             go_version="$(go_version_for_kubernetes "${CSI_PROW_WORK}/src/kubernetes" "$version")" || die "cannot proceed without knowing Go version for Kubernetes"
             # Changing into the Kubernetes source code directory is a workaround for https://github.com/kubernetes-sigs/kind/issues/1910
             # shellcheck disable=SC2046
-            (cd "${CSI_PROW_WORK}/src/kubernetes" && run_with_go "$go_version" kind build node-image --image csiprow/node:latest --kube-root "${CSI_PROW_WORK}/src/kubernetes") || die "'kind build node-image' failed"
```

Comment:
pls make changes in csi-release-tools repo first, and then make the vendor change in this repo, thx

Reply:
Oh, sorry, I didn't realize this was vendored in from another repo. The csi-release-tools PR is here: https://github.com/kubernetes-csi/csi-release-tools/pull/259

Thanks for the pointer!